app/src/main/java/mozilla/org/webmaker/fragment/WebviewFragment.java

```diff
@@ -49,7 +49,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         }
 
         // Create webview
-        mWebView = new WebmakerWebView(mView.getContext(), pageId);
+        mWebView = new WebmakerWebView(mView.getContext(), pageId, null);
```

Comment:
no need for the explicit `null` if we make sure we have a fall-through constructor for `WebmakerWebView`